01_prepare_host.sh

```diff
@@ -21,9 +21,10 @@ if [[ $OS == ubuntu ]]; then
   elif [[ ${DISTRO} == "ubuntu20" ]]; then
     sudo update-alternatives --install /usr/bin/python python /usr/bin/python3.8 1
   fi
-
-else
-  sudo dnf update -y && sudo dnf install -y epel-release
+elif [[ $OS == centos ]]; then
+  sudo dnf upgrade -y
+  sudo dnf config-manager --set-enabled powertools
+  sudo dnf install -y epel-release epel-next-release
```

Comment:
@elfosardo these changes LGTM, but could you share a bit more on why we would need to add the new epel-next repository for CentOS, on the PR descriptions if possible. Thanks!

Reply:
done